.github/workflows/gradual-strict-null-checks.yml

```diff
@@ -47,13 +47,12 @@ jobs:
               --data "{ \"body\": \"${1}\" }"
           }
 
-          sed -i 's/\/\/\s*"strictNullChecks":\s*true,/"strictNullChecks": true,/' "current/tsconfig.json"
-          sed -i 's/\/\/\s*"strictNullChecks":\s*true,/"strictNullChecks": true,/' "main/tsconfig.json"
+          YARN_1="yarn --mutex network --cwd ./current"
+          YARN_2="yarn --mutex network --cwd ./main"
 
-          yarn --mutex network --cwd ./current 2> .stderr-current > .out-current &
+          $YARN_1 install --ignore-scripts &> /dev/null && $YARN_1 build --strictNullChecks true 2> .stderr-current > .out-current &
```

Comment:
As an explanation for this change:
`install` runs the `prepare` script that transitively calls `build`. To improve performance we just want to download dependencies and then run `build`